hoot-core-test/src/test/cpp/hoot/core/algorithms/subline-matching/FrechetSublineMatcherTest.cpp

```diff
@@ -149,8 +149,8 @@ class FrechetSublineMatcherTest : public HootTestFixture
     vector<WaySublineMatch> m = uut.findMatch(map, w1, w2, score).getMatches();
     HOOT_STR_EQUALS(1, m.size());
     HOOT_STR_EQUALS(
-      "subline 1: start: way: -1 index: 87 fraction: 0 end: way: -1 index: 90 fraction: 0\n"
-      "subline 2: start: way: -2 index: 0 fraction: 0 end: way: -2 index: 4 fraction: 0",
+      "subline 1: start: way: -1 index: 0 fraction: 0 end: way: -1 index: 9 fraction: 0\n"
+      "subline 2: start: way: -2 index: 0 fraction: 0 end: way: -2 index: 20 fraction: 0",
```

Comment:
Looking at this I thought this couldn't be right but these changes did in fact make this matcher match more of the two lines.  Well done.

Reply:
Yeah, I noticed that.  Don't thank me....thank @yuansha84. :-)  Regression test scoring didn't change in any significant way and unit tests picked up about as many matches as they lost, but at least we have some more correct code now.  There was also some angle calc code in a network conflation test that got affected as well...I was a little unsure about it at first, but I think its changes are correct too.

Reply:
Yeah that took me a bit to verify that it was working correctly but everything looks good to me.